ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss

```diff
@@ -34,19 +36,49 @@ $text: map.get(gio.$mat-dove-palette, default);
   }
 }
 
+.gio-menu-selector__mat-option.mat-option.mat-selected {
+  color: #000000; // TODO: Change this to $accent or $text...
```

Comment:
We should check with @audrey-tlm what she thinks... Before it was a the green primary color. The drop-down is also always with a white background, so no need to link it to $text I think....

Reply:
I decided to hardcode it to black for now. Maybe we'll find a way to improve this later.